python/mlcroissant/mlcroissant/_src/core/data_types.py

```diff
@@ -28,6 +28,9 @@ def check_expected_type(issues: Issues, jsonld: Json, expected_type: str):
     constants.SCHEMA_ORG_DATA_TYPE_IMAGE_OBJECT: (
         constants.SCHEMA_ORG_DATA_TYPE_IMAGE_OBJECT
     ),
+    constants.SCHEMA_ORG_DATA_TYPE_AUDIO_OBJECT: (
```

Comment:
I believe this has changed. Can you rebase to have a freshest version of the code?

```
git pull --rebase origin main
```